sysutils/duplicity/Portfile

```diff
@@ -1,41 +1,40 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
 
 PortSystem          1.0
 PortGroup           python 1.0
 
 name                duplicity
 categories          sysutils
 
-version             0.7.17
+version             0.8.15          ;# released July 27, 2020.
```

Comment:
Comments should be on their own line. I'm not sure if putting them at the end of another line like this works, but it's not something we do anywhere else in MacPorts.

Reply:
Is there a specific reason why this comment is needed for this port? We don't usually add a comment with a release date.

Reply:
No, maybe I added it because it was more rescent than the one mentioned in the ticket. Removed √.